### ISSUE TITLE
Cooperate with JSCOnly port of WebKit

### ISSUE THREAD

**Issue by annulen** [dibyendumajumdar/B3#1]
Hi! As you may know, now WebKit has JSCOnly port which does not have any mandatory build dependencies except ICU. So, you don't have to maintain a fork just to build JavaScriptCore without dependencies anymore.

However, we [1] found that B3 still fails many tests on Win64 [2], and someone needs to finsh Windows port of bmalloc (https://bugs.webkit.org/show_bug.cgi?id=143310). It would be great if you could cooperate with WebKit project to make JSCOnly port better option for developers who need only JS engine instead of full WebKit.

Thanks!

[1] QtWebKit project, https://github.com/annulen/webkit/wiki
[2] You cannot run test harness without LayoutTests and Tools directories of original Webkit repo.

**Comment by dibyendumajumdar** [comment]
Hi - I was trying to create a standalone B3 - which doesn't have JavaScript but only the JIT components. I assume the JSCOnly port is not exactly the same?

**Comment by annulen** [comment]
No, they are not the same. I've missed the point about B3 without JavaScript. However, I'd like to warn you that B3 is designed to work on top of JITs of lower tiers (baseline and DFG)
